week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,8 +10,16 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(bushes) {
+// clearImmediate;
+  let bushBerryColours = bushes.every(Element => Element === "pink");
+  let message;
+  if(bushBerryColours){
```

Comment:
oooor you can do it with one line 

 return  colours.every(colour => colour === "pink") ? "Bush is safe to eat from" : "Toxic! Leave bush alone!"